<ui version="4.0" >
 <class>SessionsDlg</class>
 <widget class="QDialog" name="SessionsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MP3 Diags - Sessions</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QWidget" native="1" name="widget_2" >
       <layout class="QHBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="m_pSessionsG" >
          <property name="tabKeyNavigation" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" native="1" name="widget" >
          <layout class="QVBoxLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QTreeView" name="m_pDirectoriesT" />
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" native="1" name="widget_3" >
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="m_pOpenB" >
          <property name="text" >
           <string>&amp;Open</string>
          </property>
          <property name="default" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pNewB" >
          <property name="text" >
           <string>&amp;New ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pEditB" >
          <property name="text" >
           <string>&amp;Edit ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pSaveAsB" >
          <property name="text" >
           <string>Save &amp;as ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pEraseB" >
          <property name="text" >
           <string>E&amp;rase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pLoadB" >
          <property name="text" >
           <string>&amp;Load ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pHideB" >
          <property name="text" >
           <string>&amp;Hide</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pCancelB" >
          <property name="text" >
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pOpenLastCkB" >
     <property name="text" >
      <string>At s&amp;tartup open the last session automatically</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pSessionsG</tabstop>
  <tabstop>m_pDirectoriesT</tabstop>
  <tabstop>m_pOpenB</tabstop>
  <tabstop>m_pNewB</tabstop>
  <tabstop>m_pEditB</tabstop>
  <tabstop>m_pSaveAsB</tabstop>
  <tabstop>m_pEraseB</tabstop>
  <tabstop>m_pLoadB</tabstop>
  <tabstop>m_pHideB</tabstop>
  <tabstop>m_pCancelB</tabstop>
  <tabstop>m_pOpenLastCkB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
